stdlib/file/src/main/ballerina/src/file/Module.md

```diff
@@ -25,13 +25,13 @@ public function main() {
     string|error results = file:createDir(“foo/bar”);
     
     // Create a directory with any non existence parents.
-    string|error results = file:createDir(“foo/bar”, parentDirs = true);
+    string|error results = file:createDir(“foo/bar”, true);
```

Comment:
We should use some `io:println() lines to validate the results of each of these operations. They should appear in .out file

Reply:
sorry this comment applies to .bal file.